x/magpie/internal/keeper/keeper.go

```diff
@@ -20,6 +20,29 @@ func NewKeeper(cdc *codec.Codec, storeKey sdk.StoreKey) Keeper {
 	}
 }
 
+// -------------
+// --- Params
+// -------------
+
+// SetDefaultSessionLength allows to set a default session length for new magpie sessions.
+// The specified length is intended to be in number of blocks.
+func (k Keeper) SetDefaultSessionLength(ctx sdk.Context, length int64) {
+	store := ctx.KVStore(k.StoreKey)
+	store.Set([]byte(types.SessionLengthKey), k.Cdc.MustMarshalBinaryBare(length))
+}
+
+// GetDefaultSessionLength returns the default session length in number of blocks.
+func (k Keeper) GetDefaultSessionLength(ctx sdk.Context) int64 {
+	store := ctx.KVStore(k.StoreKey)
+
+	length := int64(0)
```

Comment:
Isn't here better putting `int64(240)`? 
So if for some reasons the if below doesn't match the condition at least we return a valid `DefaultSessionLength`.

Reply:
I put the default value to be `0` as this would make all sessions expire immediately upon being created. I think it's always better to return the default type values based on the operations they are going to be used in later (in this case the addition has the neutral value equals to `0`). 

If all the sessions expire immediately, we will notice it and fix it fast enough. Also, by putting a hardcoded `240` here would mean we would have to remember to change it here and inside the default genesis value if we would have to edit it. As we cannot import the genesis package inside the keeper (it would cause circular dependencies) I just thought `0` would be better overall.